src/main/java/executor/api/controller/ProxySourceController.java

```diff
@@ -0,0 +1,38 @@
+package executor.api.controller;
+
+import executor.api.model.ProxyConfigHolderDTO;
+import executor.api.service.impl.ProxySourceQueueHandler;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Optional;
+
+@RestController
+@RequestMapping(value = "/api/v1/proxy")
```

Comment:
let's try to use plural `proxies` :)
```suggestion
@RequestMapping(value = "/api/v1/proxies")
```